### PR TITLE
Biopet add r dependencies for reports

### PR DESCRIPTION
**Pull request by rhpvorderman**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

For the reports biopet depends on R. These dependencies are now added to the package.

### CODE REVIEW

File: recipes/biopet/0.7.0/meta.yaml [hunk 1]
```diff
@@ -15,6 +15,9 @@ requirements:
   run:
     - openjdk
     - python
+    - r
```
Comment (bgruening) [inline recipes/biopet/0.7.0/meta.yaml]: Can you use `r-base` here and in the other places?

**Comment by rhpvorderman** [comment]
I see travis is failing because of disk size: `OSError: [Errno 28] No space left on device`. Should I be more specific than r-essentials?

**Comment by bgruening** [comment]
Yes, please.

**Comment by rhpvorderman** [comment]
Done! Thanks for reviewing!

**Pull request merged**